### DISCUSSION TITLE
The calibration problem under optitrack and how to stop the motor of cf

### DISCUSSION THREAD

**Discussion by Sherry-97** [USC-ACTLab/crazyswarm#550] [Q&A]
I ran into two problems while flying with crazyflie and was hoping to get some advice.
**1.**When I use optitrack to track multiple cf flights, I run into problems with incorrect tracking between tracked targets, such as cf #1 being thought to be #2. I tried to put calibration points in different positions of the cf, but when the number of cf goes up, the number of calibration points is not enough to differentiate, is there any other good way to do it?
**2.**In addition, cf may encounter the problem of side flip, when the plane's motor will rotate rapidly and make a big sound on the ground, which is more dangerous. Is there some way to make the cf's motor stop at this time by the program. since the command to turn off the operation is useless at this time.

**Comment by whoenig** [comment]
1. Use libobjecttracker, rather than your motion capture software for tracking.
2. The latest official firmware release stops the motors automatically after any flip (and stays in an emergency mode). If you don't see this behavior, you are perhaps not running a recent firmware version?

**Reply by Sherry-97** [reply]
Thank you very much for your reply.
I was wondering when I use libobjecttracker for tracing, what is the difference between this and motion capture, in other words, is this approach sacrificing performance in other areas to ensure the accuracy of tracing?
Also, I do use the latest official firmware, and it is true that the motor stops spinning when the plane crashes normally, however, sometimes it appears that the motor does not stop, but instead spins faster than before, as if it has received the command to take off and spins quickly to achieve an ascent? (although I did not give the command, but stopped the running program)

**Reply by whoenig** [reply]
A discussion of the different modes is available at https://crazyswarm.readthedocs.io/en/latest/configuration.html#select-object-tracking-mode-motion-capture-only.

Motor: this sounds more like a firmware issue - you might be able to get help in the official forum if you can reproduce the problem.
